docs/content/querying/multi-value-dimensions.md

```diff
@@ -4,7 +4,21 @@ layout: doc_page
 # Multi-value dimensions
 
 Druid supports "multi-value" string dimensions. These are generated when an input field contains an array of values
-instead of a single value (e.e. JSON arrays, or a TSV field containing one or more `listDelimiter` characters).
+instead of a single value (e.e. JSON arrays, or a TSV field containing one or more `listDelimiter`/`multiValueDelimiter` characters).
+The `multiValueDelimiter` is a map of field->value pairs, it can specify the delimiter of each dimensions.
```

Comment:
Why "field"? Maybe "column name" or "dimension name"?

Reply:
"each dimension"

Reply:
Thanks, dimension->delimiter pairs is more clear